tests/adiabatic_initial_conditions_chunk/screen-output

```diff
@@ -5,31 +5,31 @@ Number of degrees of freedom: 1,444 (882+121+441)
 *** Timestep 0:  t=0 seconds
    Solving temperature system... 0 iterations.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 26+0 iterations.
+   Solving Stokes system... 25+0 iterations.
```

Comment:
why does this file change? the test uses a spherical shell boundary, but I thought this patch should only apply to the chunk geometry, right?